_includes/head.html

```diff
@@ -33,7 +33,6 @@
   {% endif %}
 
 
-  <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
-  <link rel="stylesheet" href="{{ '/css/main.css' | prepend: site.baseurl }}">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />  <link rel="stylesheet" href="{{ '/css/main.css' | prepend: site.baseurl }}">
```

Comment:
Nit: Could we keep these on separate lines

```suggestion
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />  
  <link rel="stylesheet" href="{{ '/css/main.css' | prepend: site.baseurl }}">
```

Reply:
Woops yeah that was a mistake! [fbfa445](https://github.com/srobo/website/pull/434/commits/fbfa445a7a23c8ef9934ccc61d0a9901d1a3b48f)